cli/index.js

```diff
@@ -8,80 +8,82 @@ const { mochify } = require('@mochify/mochify');
 const opts = yargs(hideBin(process.argv))
   .usage(
     '$0 [options] <spec...>',
-    'Run Mocha tests using Mochify.',
+    'Run Mocha tests in real browsers.',
     (local) => {
       local
         .example(
           '$0 --driver puppeteer --bundle browserify "./src/**/*.test.js" ',
           'Bundle all files matching the given spec using browserify and run them using @mochify/driver-puppeteer.'
         )
         .example(
-          '$0 --config package.json "./src/**/*.test.js" ',
-          'Run all tests matching the given spec using the configuration from package.json\'s "mochify" key.'
+          '$0 --esm --reporter dot --driver puppeteer "./src/**/*.test.js" ',
+          'Run all tests matching the given spec as ES modules in puppeteer and use the "dot" reporter for output.'
         )
         .example(
-          '$0 --esm --reporter dot "./src/**/*.test.js" ',
-          'Run all tests matching the given spec as ES modules and use the "dot" reporter for output.'
+          '$0 "./src/**/*.test.js" ',
+          'Run all tests matching the given spec using the configuration from package.json'
```

Comment:
If there is a `mochify` key in `package.json`. cosmiconfig tries various options in a specific order. Maybe we should just document that with a link to cosmiconfig?

See https://github.com/davidtheclark/cosmiconfig#explorersearch

Reply:
Ah ok, that's not entirely obvious tbh (I remember seeing a list of filenames being removed in a commit once and thought this meant there is no default lookup happening, but apparently that's not the case). Will adjust this accordingly.

Reply:
Yes, I removed the file list because I discovered that I replicated the cosmiconfig defaults 😆